tasks/main.yml

```diff
@@ -1,4 +1,9 @@
 ---
+- name: check if gluster_repos_activationkey is set
+  fail:
+    msg: "gluster_repos_activationkey is needed for enabling the repositories"
+  when: gluster_repos_activationkey is not defined
```

Comment:
shouldn't this be checked only for tags:activationkey?

Reply:
fixed. added conditional comparison between username, password and activation_key.